test/e2e/performanceprofile/testdata/render-expected-output/bootstrap/extra-mcp/openshift-bootstrap-master_kubeletconfig.yaml

```diff
@@ -2,9 +2,9 @@ apiVersion: machineconfiguration.openshift.io/v1
 kind: KubeletConfig
 metadata:
   creationTimestamp: null
-  name: performance-openshift-bootstrap-master
   labels:
-    performance.openshift.io/weak-owner-reference-name: "openshift-bootstrap-master"
+    performance.openshift.io/weak-owner-reference-name: openshift-bootstrap-master
```

Comment:
looks like we are getting a lot of changes not really related to the PR but to the render sync.  could it be environment specific invocation of render sync ? or maybe we just forgot to submit these generate files (that did not fail the tests so far)

Reply:
Yes, I ran `make render-sync` on my station. Should I remove all these?

Reply:
if its not aligned with 4.16 (lets double check) then remove them . otherwise we can keep them